### PR TITLE
Update moby-runc to version 1.0.0~rc94+azure

### PR DESCRIPTION
**Pull request by anphel31**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Update moby-runc to latest version

###### Change Log  

- Update moby-runc SPEC to use latest
- Update moby-runc dependencies to latest versions (golang-sys, golang-crypto, logrus, urfave-cli)
- Update cgmanifest.json with dependency packages listed above
- Add generate-sources.sh script to create moby-runc-v1.0.0-rc94.tar.gz (latest code does not build without .git)

###### Does this affect the toolchain?  

NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Local package build

### CODE REVIEW

**Review by jslobodzian — APPROVED** [review]

File: SPECS/moby-runc/moby-runc.spec [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+Version:        1.0.0~rc94+azure
+Release:        1%{?dist}
 License:        ASL 2.0
 URL:            https://runc.io/
 
-#Source0:       https://github.com/opencontainers/runc/releases/download/v1.0.0-rc10/runc.tar.xz
-Source0:        runc-v1.0.0-rc10.tar.xz
-#Source1:       https://github.com/sirupsen/logrus/archive/v1.4.1.tar.gz
-Source1:        logrus-v1.4.1.tar.gz
+# See generate-sources.sh for creating runc source tarball
```
Comment (PawelWMS) [inline SPECS/moby-runc/moby-runc.spec:8]: What do you think about making `RUNC_COMMIT` and `VERSION` input arguments for `generate-sources.sh` and providing the argument values this tarball was generated with in this comment? This way we'd only have to update the spec for a version update and won't be risking keeping an outdated script.

File: SPECS/moby-runc/moby-runc.spec [hunk 2]
```diff
@@ -1,22 +1,23 @@
 Summary:        CLI tool for spawning and running containers per OCI spec.
 Name:           moby-runc
-Version:        1.0.0~rc10+azure
-Release:        6%{?dist}
+Version:        1.0.0~rc94+azure
```
Comment (jslobodzian) [inline SPECS/moby-runc/moby-runc.spec:3]: Is this what the moby upstream team is supporting?

Comment (anphel31) [reply SPECS/moby-runc/moby-runc.spec:3]: Yes this is the latest

File: SPECS/moby-runc/moby-runc.spec (hunk 1 above)
Comment (anphel31) [reply SPECS/moby-runc/moby-runc.spec:8]: All of the moby-* packages have the same type of script, so if we do this I think it should be for all of them at the same time. Will file a bug to track this, but not as part of this PR.

**Pull request merged**